src/CMakeLists.txt

```diff
@@ -330,36 +330,24 @@ if(WITH_BREAKPAD)
   set(breakpad_SOURCE_DIR ${CMAKE_SOURCE_DIR}/src/breakpad)
   set(lss_SOURCE_DIR ${CMAKE_SOURCE_DIR}/src/lss)
 
-  add_custom_target(breakpad_lss_symlink)
-  add_custom_command(
-    TARGET breakpad_lss_symlink
-    PRE_BUILD
-    COMMAND ${CMAKE_COMMAND} -E create_symlink ${lss_SOURCE_DIR} ${breakpad_SOURCE_DIR}/src/third_party/lss
-    COMMENT "Creating symbolic link lss -> breakpad third party"
-  )
   ExternalProject_Add(
     breakpad_project
     SOURCE_DIR "${breakpad_SOURCE_DIR}"
-    CONFIGURE_COMMAND cd "${breakpad_SOURCE_DIR}"
-        COMMAND "${breakpad_SOURCE_DIR}/configure"
-	    "CC=${CMAKE_C_COMPILER}"
-	    "CXX=${CMAKE_CXX_COMPILER}"
-	    "CFLAGS=${CMAKE_C_FLAGS} -fPIC -Wno-unknown-warning-option -Wno-array-bounds  -Wno-maybe-uninitialized"
-	    "CXXFLAGS=${CMAKE_CXX_FLAGS} -fPIC -Wno-unknown-warning-option -Wno-ignored-qualifiers  -Wno-array-bounds  -Wno-maybe-uninitialized"
-            "LDFLAGS=${CMAKE_EXE_LINKER_FLAGS} -fPIC -Wno-unknown-warning-option -Wno-array-bounds  -Wno-maybe-uninitialized"
-    BUILD_COMMAND
-    /bin/sh -cx "cd ${breakpad_SOURCE_DIR} && make"
+    PATCH_COMMAND ${CMAKE_COMMAND} -E create_symlink ${lss_SOURCE_DIR} ${breakpad_SOURCE_DIR}/src/third_party/lss
+    CONFIGURE_COMMAND "${breakpad_SOURCE_DIR}/configure"
+            "CXXFLAGS=${CMAKE_CXX_FLAGS} -Wno-array-bounds -Wno-maybe-uninitialized -fPIC"
+            "LDFLAGS=${CMAKE_EXE_LINKER_FLAGS}"
```

Comment:
This needs 

"CC=${CMAKE_C_COMPILER}"
"CXX=${CMAKE_CXX_COMPILER}"

and -fPIC -Wno-unknown-warning-option -Wno-ignored-qualifiers

Reply:
@irq0, thank you for your review. yeah, i missed the `-fPIC` flag when rebasing my change. added it back.

however, if we drop the compile flags from `LDFLAGS`, `-Wno-unknown-warning-option` is not necessary anymore. as the commit message explains, we should not pass link flags to compiler.